libs/category/src/category-state.module.ts

```diff
@@ -4,11 +4,15 @@ import { EffectsModule } from '@ngrx/effects';
 
 import { daffCategoryReducers } from './reducers/category-reducers';
 import { DaffCategoryEffects } from './effects/category.effects';
+import { DaffDefaultCategoryPageSize } from './resolvers/public_api';
 
 @NgModule({
   imports: [
     StoreModule.forFeature('category', daffCategoryReducers),
     EffectsModule.forFeature([DaffCategoryEffects]),
-  ]
+	],
+	providers: [
+		{ provide: DaffDefaultCategoryPageSize, useValue: 12 }
```

Comment:
Not saying that you should do this, but default values can be passed to the injection token constructor. Just FYI in case you find that a better way to provide default token values.

Reply:
I'm good with as is